### PR TITLE
DOCS-1076 - Documenting PGD 5.6.1

### PR DESCRIPTION
**Pull request by djw-m**
Near final PR for 5.6.1 changes.

### CODE REVIEW

**Comment by djw-m** [comment]
Note that this will currently show a bad link in the tests as there is a single link to an EPAS 17 (as yet unreleased) page.

File: product_docs/docs/pgd/5.6/reference/functions.mdx [hunk 1]
```diff
@@ -282,17 +281,60 @@ If a slot is dropped concurrently, the wait ends for that slot.
...
--   `target_lsn` &mdash; LSN to wait for or, if NULL, use the current write LSN on the
-    local node.
+| Parameter    | Description                                                                 |
+|--------------|-----------------------------------------------------------------------------|
+| `node_name`  | Name of the node to wait for. If NULL, waits for all nodes.                 |
+| `target_lsn` | LSN to wait for. If NULL, uses the current wal_flush_lsn on the local node. |
+
+#### Notes
+
+Requires `bdr_superuser` privileges to use.
```
Comment (sumarjee) [inline product_docs/docs/pgd/5.6/reference/functions.mdx:336]: Please change this to bdr_application

File: product_docs/docs/pgd/5/reference/index.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+<<<<<<< HEAD
```
Comment (sumarjee) [inline product_docs/docs/pgd/5/reference/index.json:2]: Not sure if this is expected, seems like some unresolved conflict

File: product_docs/docs/pgd/5.6/reference/functions.mdx (hunk 1 above)
Comment (djw-m) [inline product_docs/docs/pgd/5.6/reference/functions.mdx:336]:
```suggestion
Requires `bdr_application` privileges to use.
```

**Review by mwanner — APPROVED** [review]
LGTM

**Pull request merged**